src/Compilers/CSharp/Portable/Symbols/Source/SourceCustomEventAccessorSymbol.cs

```diff
@@ -29,7 +29,7 @@ internal sealed class SourceCustomEventAccessorSymbol : SourceEventAccessorSymbo
             DiagnosticBag diagnostics)
             : base(@event,
                    syntax.GetReference(),
-                   syntax.Body?.GetReference(),
+                   syntax.Body?.GetReference() ?? syntax.ExpressionBody?.GetReference(),
```

Comment:
Yes, there are a number of tests that hit that case -- in fact, due to the change in binding to just consider whether or not `bodySyntax` is non-null, this hits *every* test case that binds an expression bodied event.